volo-grpc/src/codec/decode.rs

```diff
@@ -28,7 +28,7 @@ use crate::{
 ///
 /// Provides an interface for receiving messages and trailers.
 pub struct RecvStream<T> {
-    body: hyper::Body,
+    body: surf::Body,
```

Comment:
so I don't know why this isn't using volo_grpc::Body, but it seems like a natural place that it should be used